GUI/GUIComboBox.cpp

```diff
@@ -335,12 +337,11 @@ void GUIComboBox::ReceiveSignal(GUIPanel *Source, int Code, int Data)
             return;
         }
     
-        // Click
         int mouseX = 0;
         int mouseY = 0;
         m_Manager->GetInputController()->GetMousePosition(&mouseX, &mouseY);
-        // Unless we check for the mouse not being over the otehr components, this causes irritating accidental closing of the list when clicking on the textbox and button
-        if (Code == GUIListPanel::Click && !m_TextPanel->PointInside(mouseX, mouseY) && !m_Button->PointInside(mouseX, mouseY))
+		// Mouse down anywhere outside the list panel.
+        if (Code == GUIListPanel::Click)
```

Comment:
I'm sure the answer is no, but I want to be sure, this doesn't disallow you from selecting options in the combobox option list or anything, right?

Reply:
right. selecting an option from the list sends a different code, mousedown instead of click.